dotCMS/src/main/webapp/html/js/editor-js/src/lib/editor/sdk-editor-vtl.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Do you know if this does something?

Reply:
We need to set the build in nx so it doesn't emit declation files or anything else... it should be one file.